### ISSUE TITLE
Activity breaks when playing synced content on MS Store Plex

### ISSUE THREAD

**Issue by pegasy** [Tautulli/Tautulli-Issues#155]
**Version:** V2.1.26

**Branch:** master

**Commit hash:** aa023f016671e78463dd95e8f662cbc6cc712cf0

**Operating system:** `Linux 4.18.20-unRAID (#1 SMP Fri Nov 23 11:38:16 PST 2018)`

**Python version:** 2.7.15 (default, Aug 16 2018, 14:17:09) [GCC 6.4.0]

**What you did?**

Go to the homepage when someone is playing locally synced stuff on Plex (from the MS store)

**What happened?**

![](https://i.imgur.com/xoEUkdV.png)

**What you expected?**

Show the activity, right now it doesn't show *anyone* playing if one of them is using the MS Store Plex and doing synced content, [this is how it looks in the plex dashboard](https://i.imgur.com/JW1XtqB.png).
We are both using the MS Store Plex, Kaderin has stuff synced locally with the [settings below](https://i.imgur.com/rsqWUk9.png), I (Pegasy) is direct playing. As soon as Kaderin stops playing or plays something thats not synced to her PC Tautulli starts working again. 

**How can we reproduce your issue?**
Download plex from the MS store, sync something 

**What are your (relevant) settings?**

Don't think any settings on tautulli affects this.
[These are the settings on the MS Store Plex app](https://i.imgur.com/rsqWUk9.png)

**Link to logs:** [Gist](https://gist.github.com/pegasy/5a4dca72a99c0d90842a32fd797dba85)

**Comment by JonnyWong16** [comment]
I just tried this, but I cannot reproduce the issue.

![image](https://user-images.githubusercontent.com/9099342/50728593-49999a00-10e1-11e9-898f-18c60310c89e.png)

Can you follow the instructions here to grab your `sessions.xml` when the issue occurs?

https://github.com/Tautulli/Tautulli-Wiki/wiki/Debugging

**Comment by pegasy** [comment]
[Here you go](https://gist.github.com/pegasy/a48b3efd5fa3d357bca15defd14fe7b5).  
I replaced the user id / guid and ip adresses and only included the part that makes it error out. 

After reading through the xml myself I realize that I missed to mention another step to replicate.  
The data that is synced was first optimized on the server.  
1. Optimize media (I did android, 8mbit 1080p and named it Android)
2. Sync to Plex for Windows using 8 mbit preset or whatever its called
3. Play stuff in Plex for Windows
